alembic/versions/0db31a429322_create_algorithm_tables.py

```diff
@@ -0,0 +1,51 @@
+"""create algorithm tables
+
+Revision ID: 0db31a429322
+Revises: 6052c193a26a
+Create Date: 2024-09-19 11:45:31.232579
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '0db31a429322'
+down_revision: Union[str, None] = '6052c193a26a'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('algorithm',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('is_default', sa.Boolean(), nullable=False),
+    sa.Column('label', sa.String(length=255), nullable=False),
+    sa.Column('description', sa.String(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('label')
+    )
+    op.create_table('algorithm_pass',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('algorithm_id', sa.Integer(), nullable=False),
+    sa.Column('blocking_keys', sa.JSON(), nullable=False),
+    sa.Column('evaluators', sa.JSON(), nullable=False),
```

Comment:
#13 indicates that the `blocking_keys` column should be list of int and that the `evaluators` column should be list of str. What was the reason for the change to JSON type?